lib/scrabble.rb

```diff
@@ -9,5 +8,47 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+
+
+  def score(letters)
+    if letters == nil
+      return 0
+    else
+    new_letters = letters.upcase.chars
+    valuator =  new_letters.map do |x|
+                LIB[x]
+                end
+    return valuator.sum
   end
+  end
+
+  def multipliers(word, multiplier, bonus = 1)
+    if word.length < 7
+    new_letters = word.upcase.chars
+    valuator =  new_letters.map do |x|
+                LIB[x]
+                end
+    multiply =  valuator.zip(multiplier).map{|x,y| x * y}
+
+   return  (multiply.sum) * bonus
+
+    elsif
+      new_letters = word.upcase.chars
+      valuator =  new_letters.map do |x|
+                  LIB[x]
+                  end
+    seven_letter_bonus =  valuator.zip(multiplier).map{|x,y| x * y}
+    multiply = (seven_letter_bonus.sum) + 10
+
+    return(multiply) * bonus
+  end
+  end
```

Comment:
Looks good! Check your indentation. Also, seems like there may be some duplication here. See if you can limit the conditional regarding the length of the word to only that place where it comes into play as a refactor.